drivers/serial/Kconfig

```diff
@@ -83,6 +90,15 @@ config UART_DRV_CMD
 
 	  Says no if not sure.
 
+config UART_WIDE_DATA
+	bool "Enable API to support data longer than 8-bit"
+	help
+	  This enables the API to process data longer than 8-bit.
+	  This is up to the driver to implement the necessary functions
+	  to properly support this.
+
+	  Says no if not sure.
```

Comment:
If we want to keep recommendation like this i would use something like "Disable if not sure"

Reply:
Since default is disabled, this would rather be "keep disabled if not usre"

Reply:
This is not necessary, so removed.